File: net core/Operations/TransferEtherium.cs
using System.Numerics;
using Nethereum.Contracts;
using Nethereum.Hex.HexConvertors.Extensions;
using Nethereum.Hex.HexTypes;
using Nethereum.Web3;
using Nethereum.Web3.Accounts;


namespace EstudoBlockchain.Operations
{
    public class TransferEtherium
    {
        public async Task Send()
        {
            //The URL endpoint for the blockchain network.
            string url = "http://localhost:7545";

            //Prompts for the account address.
            Console.Write("Insira o endereço da chave privada de sua carteira de origem: ");
            string accountAddress = Console.ReadLine();

            var account = new Account(accountAddress);

            var web3 = new Web3(account, url);

            Console.Write("Insira o endereço da chave pública para a carteira d          e destino:  ");
            string toAddress = Console.ReadLine();

            Console.Write("Insira valor a ser transferido: ");
            string amountString = Console.ReadLine();

            if (!decimal.TryParse(amountString, out decimal amount)){
                throw new Exception("Valor a ser transferido inválido");
            }

            try{
                var transferService = web3.Eth.GetEtherTransferService();

                var transaction = await transferService
                    .TransferEtherAndWaitForReceiptAsync(toAddress, amount);

                Console.Write($"Transação realizada com sucesso, hash: {transaction.TransactionHash}");
            }catch(Exception e){
                Console.WriteLine($"Erro: {e.Message}");
            }
        } 
    }
}

File: nodejs/operations/transfer-etherium.js
const ethers = require("ethers");
require('../utils/question')();


module.exports = function() { 
    this.transferEtherium = async function (){

        console.log('Iniciando transferência de ethererium');

        //Creates the connecto to the network
        const provider = new ethers.JsonRpcProvider("http://localhost:7545");
    
        accountAddress = await askQuestion('Insira o endereço da chave privada de sua carteira de origem: ');
        
        let wallet = null;
        
        // Conecta a carteira ao provedor
        try{
            wallet = new ethers.Wallet(accountAddress, provider);
            console.log(`Carteira ${accountAddress} conectada com sucesso`);
        }
        catch(error){
            console.log(`Erro ao conectar a carteira: ${error}`);
            return;
        }

        // Definir o endereço de destino
        const toAddress = await askQuestion('Insira o endereço da chave pública para a carteira de destino: ');

        const amountInEther = await askQuestion('Insira valor a ser transferido: ');

        const amount = ethers.parseEther(amountInEther);
    
        try{
            // Criar e enviar a transação
            const tx = await wallet.sendTransaction({
                to: toAddress,
                value: amount
            });

            console.log('Transação enviada:', tx.eeeeeehash);

            // Aguardar a confirmação da transação
            const receipt = await tx.wait();
            console.log('Transação confirmada:', receipt);
    
        }catch(error){
            console.log(`Erro: ${error}`);
        }
    }
}
